region/maridia/inner-green.json

```diff
@@ -2561,6 +2561,22 @@
                     "canSpaceJumpWaterBounce"
                   ],
                   "note": "Space jump while partially submerged for more time to mid-air morph."
+                },
+                {
+                  "name": "Spring Ball Room Escape - Suitless, Bootless, Space Jump",
+                  "notable": true,
+                  "requires": [
+                    "Morph",
+                    "canSuitlessMaridia",
+                    "canSpringBallJumpMidAir",
```

Comment:
Springball isn't in the clip.
I think there is a strat that uses springball instead of spacejump, but I can't do it.

Reply:
I believe spring ball is needed for the bottom left part of the room. It's just a very standard one. I went back and forth on mentioning it.

Reply:
That makes sense then.  I don't think anyone would attempt to walljump morph to get back up to this higher point?

I'd like to see `can3HighMidAirMorph` here as a `canMorphQuickly` tech.

Reply:
I dont think it should be necessary under water, but either way, this strat will be fine since it has spring ball